src/containers/NFT/NFTHeader/index.js

```diff
@@ -0,0 +1,214 @@
+    return (
+      <table className="token-table">
+        <tbody>
+          {minted && (
+            <tr className="row">
+              <td className="col1">Minted</td>
+              <td className="col2">{minted}</td>
+            </tr>
+          )}
+          {domain && (
+            <tr className="row">
+              <td className="col1">{t('domain')}</td>
+              <td className="col2">
+                <a
+                  href={`https://${domain}`}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {domain}
+                </a>
+              </td>
+            </tr>
+          )}
+          {emailHash && (
+            <tr className="row">
+              <td className="col1">{t('email_hash')}</td>
+              <td className="col2">
+                {abbrvEmail}
+                <CopyToClipboard className="copy" text={emailHash} />
+              </td>
+            </tr>
+          )}
+          <tr className="row">
+            <td className="col1">Taxon ID</td>
+            <td className="col2">{NFTTaxon}</td>
+          </tr>
+          {uri && (
+            <tr className="row">
+              <td className="col1">URI</td>
+              <td className="col2">
+                <a href={uri} target="_blank" rel="noopener noreferrer">
+                  {abbrvURI}
+                </a>
+              </td>
+            </tr>
+          )}
+          <tr className="row">
+            <td className="col1">Transfer Fee</td>
+            <td className="col2">{transferFee}</td>
+          </tr>
+        </tbody>
+      </table>
+    )
+  }
+
+  const renderSettings = () => {
+    const { flags } = data
+
+    const burnable = flags?.includes('lsfBurnable') ? 'enabled' : 'disabled'
+    const onlyXRP = flags?.includes('lsfOnlyXRP') ? 'enabled' : 'disabled'
+    const trustLine = flags?.includes('lsfTrustLine') ? 'enabled' : 'disabled'
+    const transferable = flags?.includes('lsfTransferable')
+      ? 'enabled'
+      : 'disabled'
+    return (
+      <table className="token-table">
+        <tbody>
+          <tr className="row">
+            <td className="col1">Burnable</td>
+            <td className="col2">{burnable}</td>
+          </tr>
+          <tr className="row">
+            <td className="col1">Only XRP</td>
+            <td className="col2">{onlyXRP}</td>
+          </tr>
+          <tr className="row">
+            <td className="col1">Trust Line</td>
+            <td className="col2">{trustLine}</td>
+          </tr>
+          <tr className="row">
+            <td className="col1">Transferable</td>
+            <td className="col2">{transferable}</td>
+          </tr>
+        </tbody>
+      </table>
+    )
+  }
+
+  const renderHeaderContent = () => {
+    const { issuer } = data
+    const abbrvIssuer =
+      issuer?.length > 23 ? issuer?.slice(0, 23).concat('...') : issuer
```

Comment:
you definitely do not need the `?.` guard

Reply:
also the issuer length is always > 23. https://xrpl.org/accounts.html#addresses

Reply:
Also truncation should happen via css.

Reply:
also - _why_ are we doing this truncation? it's pretty critical to know the full issuer address. i added a comment to you and natalia on figma WRT topic